src/Git.hs

```diff
@@ -326,20 +323,19 @@ callGit userConfig args = do
 -- Interpreter for the GitOperation free monad that starts Git processes and
 -- parses its output.
 runGit
-  :: forall m a
-   . MonadIO m
-  => MonadLogger m
+  :: forall es a
+  .  (IOE :> es, MonadLoggerEffect :> es)
   => UserConfiguration
   -> FilePath
-  -> GitOperationFree a
-  -> m a
-runGit userConfig repoDir operation =
+  -> Eff (GitOperation : es) a
+  -> Eff es a
+runGit userConfig repoDir  =
```

Comment:
Here is the first runner. It "interprets" the `GitOperation` effect, using the IO effect (`IOE`) and the `MonadLoggerEffect` (we'll get to that later).